README.md

```diff
@@ -21,27 +21,20 @@ import time
 # serial (or USB over serial) device connected to Velbus controller
 port = "/dev/ttyACM0"
 
-connection = velbus.VelbusUSBConnection(port)
-controller = velbus.Controller(connection)
-controller.subscribe(_on_message)
```

Comment:
move this behind the next line

Reply:
Is the subscribe function still useful in the example? As no scan has been done and no modules have been identified, I think all activity on the bus is seen as unrecognized commands. Therefor the _on_message function is never called, I suppose.

Reply:
the on message of the controller will be called, the on_message of the corresponding module will not be called